core/scripts/cli/sponsor/create.js

```diff
@@ -0,0 +1,42 @@
+const inquirer = require("inquirer");
+const getDefaultAccount = require("../wallet/getDefaultAccount");
+
+const create = async (web3, artifacts, emp) => {
+  const ExpandedERC20 = artifacts.require("ExpandedERC20");
+  const { toWei, fromWei, toBN } = web3.utils;
+
+  const sponsorAddress = await getDefaultAccount(web3);
+
+  const scalingFactor = toBN(toWei("1"));
+  // TODO: Understand why we need a .rawValue in one case but not the other.
+  const totalPositionCollateral = toBN((await emp.totalPositionCollateral()).rawValue.toString());
+  const totalTokensOutstanding = toBN((await emp.totalTokensOutstanding()).toString());
+  if (totalTokensOutstanding.isZero()) {
+    console.log("Error: can't create initial position with this tool");
+  }
+  const gcr = totalPositionCollateral.mul(scalingFactor).divRound(totalTokensOutstanding);
+
+  const input = await inquirer.prompt({
+    message: "How many tokens to create?",
+    name: "tokensCreated"
+  });
+  const tokens = toWei(input["tokensCreated"]);
+  const collateralNeeded = toBN(tokens)
+    .mul(gcr)
+    .divRound(scalingFactor);
+  console.log("You'll need", fromWei(collateralNeeded), "ETH to borrow tokens");
```

Comment:
Here you say `ETH`. Are we assuming it's always ETH? I thought we would have ETH (in WETH) and DAI in the first product launch. You could pull the token symbol from the collateral token from the within the `emp` in a few more calls if you want to make this dynamic.

Reply:
I'm assuming it's ETH.